### ISSUE TITLE
Crash after pressing Join button

### ISSUE THREAD

**Issue by malev01ence** [radistmorse/PEPatch#3]
The game crashes after pressing Join button. Attaching log file.

[output_log.txt](https://github.com/radistmorse/PEPatch/files/5232369/output_log.txt)

**Comment by radistmorse** [comment]
Judging by the logfile, you are not using the latest version of the patch. Is there some particular reason for that?

For the older version you need to enter the hostname or IP in the server search input. Did you do it?

**Comment by malev01ence** [comment]
That's strange, I downloaded the last version from here, and so did my friend, and we both get that crash error when we press Join button. It crashes even if I try to join my own server or even when I press Join without anything selected.
I tried searching for IP too, but didn't find anything.

Hmm here's what I did: I copied the contents of PEPatch to the game directory and contents of PEServerPatch to Server folder, then I launch PE_Server.exe. Are there any other actions I needed to do?
